Description: Remove python-* available packages from requirements
We want to make use of python-* packages available upstream,
so we remove these from the requirements.txt and rely on site-packages
being available in the shipped venv.
---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- conn-check-1.0.13.orig/requirements.txt
+++ conn-check-1.0.13/requirements.txt
@@ -1,9 +1,4 @@
 characteristic >= 14.0
 cryptography >= 0.5
-pyasn1
-pyOpenSSL
 service_identity
-requests >= 2.0
-Twisted
 txrequests >= 0.9
-PyYAML
